### DISCUSSION TITLE
sourcing config

### DISCUSSION THREAD

**Discussion by mvolkmann** [nushell/nushell#3164] [Q&A]
After making a configuration change, is there a way to activate the change without exiting the current Nushell session and starting a new one? For example, if I enter `config set prompt "echo 'nu> '"`, it seems I need to exit nu and start a new session to see the change to the prompt.

**Comment by mvolkmann** [comment]
I thought perhaps the answer was `config load $(config path)`, but that doesn't work. For example, if I change the setting for `prompt` and run that, the prompt does not change. But if I exit and start a new Nushell then it does change.

**Comment by fdncred** [comment]
Some changes rely on a nushell restart. Others do not. We don't have a good model for this right now.
